pkg/plugins/repo/client.go

```diff
@@ -233,7 +236,7 @@ func (c *Client) createReq(ctx context.Context, url *url.URL, compatOpts CompatO
 		req.Header.Set("grafana-origin", orig.(string))
 	}
 
-	if c.grafanaComAPIToken != "" {
+	if strings.HasPrefix(url.String(), c.grafanaComAPIURL) && c.grafanaComAPIToken != "" {
```

Comment:
Any suggestions here?

Reply:
I agree it's a bit weird but I think it looks good for the time being :+1: